closed/src/java.base/share/classes/openj9/internal/criu/NativePRNG.java

```diff
@@ -181,11 +215,12 @@ private RandomIO(File seedFile, File nextFile) throws IOException {
             }
             this.seedIn = seedStream;
             this.nextIn = nextStream;
+            this.nextBuffer = new byte[bufferSize];
         }
 
-        // Read data.length bytes from in.
-        // These are not normal files, so we need to loop the read.
-        // Just keep trying as long as we are making progress.
+        // read data.length bytes from in
+        // these are not normal files, so we need to loop the read
+        // just keep trying as long as we are making progress
```

Comment:
I think this was better as three sentences.

Reply:
updated